tests/integration/targets/win_dns_zone/defaults/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
You can probably just remote this file and dir as it doesn't seem like it's needed.